tests/dialects/test_bigquery.py

```diff
@@ -6,6 +6,10 @@ class TestBigQuery(Validator):
     dialect = "bigquery"
 
     def test_bigquery(self):
+        self.validate_identity("SELECT * FROM a-b-c")
```

Comment:
Can we also add an example with multiple hyphens in combination with tokens of type TokenType.DOT?

```sql
SELECT * FROM a-b-c.mydataset.mytable
```

Reply:
done